### ISSUE TITLE
Cannot add certificates to new Classic VM's via Add-AzureProvisioningConfig

### ISSUE THREAD

**Issue by smaine** [Azure/azure-powershell#5560]
### Description
The -Certificates property of Add-AzureProvisioningConfig no longer has any effect and cannot be used to provision new Classic VM's with certificates. This is a functional regression from previous versions of the Azure Powershell SDK.

The AutoMapper Update in commit [37825fa](https://github.com/Azure/azure-powershell/commit/37825fa4574c5fa8cad604f1135249793d50c545#diff-90282ac36665778677b6f252402e6adc) added an explict Ignore mapping for StoredCertifcateSettings on line 269 of ServiceManagementProfile.cs. I think this is responsible for the break. Is it a problem to add it back?

### Script/Steps for Reproduction

1) Add some certificates to a cloud service
2) Create a new VM configuration that references those certificates via something like the following:

```
$vm = New-AzureVMConfig -ImageName $imageName -Name $name -InstanceSize "Medium" -HostCaching ReadWrite -AvailabilitySetName $availabilitySetName
              Add-AzureProvisioningConfig -Windows -AdminUsername $adminUserName -Password $defaultPassword  -Certificates $certificates 

$vm | New-AzureVM -ServiceName $serviceName

```
3) Observe that once the VM gets provisioned, the certificates are not on the VM (they aren't they because the role config doesn't reference any StoredCertificates, due the above bug)

**Comment by smaine** [comment]
Any updates on this issue? It's blocking our ability to move to new versions of the Azure Powershell cmdlets. I've already highlighted the change that broke the scenario; can anyone provide context on why it was removed and what the workaround is?

**Comment by cormacpayne** [comment]
This has been fixed in version 5.2.0 of `Azure`
